docs/content/querying/topnquery.md

```diff
@@ -48,7 +48,7 @@ A topN query object looks like:
   "postAggregations": [
     {
       "type": "arithmetic",
-      "name": "sample_divide",
+      "name": "average",
```

Comment:
Why "average"? It divides sum by field "some_metrics" with sum by field "count".